routes/assets-route.js

```diff
@@ -34,4 +35,15 @@ router.post('', upload.array('images', MAX_IMAGE_COUNT), async (req, res) => {
   res.json(response);
 });
 
+/**
+ * Generate the signed url for S3 bucket for an asset.
+ */
+router.get('/:key', async (req, res) => {
```

Comment:
Is it commonplace for these args to be hardcoded strings like this?

Reply:
Yeah. that's the standard. You want to be able to read the route all together, so making it a string you have to look up would make it hard.

Reply:
np, starting to read up on express and yeah, looks pretty standard. Thx for verifying.